release-highlights/release-highlights-2021.11/release-highlights.md

```diff
@@ -0,0 +1,61 @@
+# PatternFly Release Highlights
+## Release 2021.11
+----------------------------------------------------------
+## New features and enhancements
+
+### [Tree-view enhancements](https://www.patternfly.org/v4/components/tree-view)
+
+![compact tree view with guides](./img/tree-view-compact.png)
+
+Adds new variants of the tree view component that are better designed to work with dense content and confined spaces. These include two compact variants and the ability to add guides to any tree view to better visualize the hierarchy. We have also made the toolbar/header area flexible to pass in anything you want including a search input, actions, or anything that makes sense.
+
+### [Search input with submit button](https://www.patternfly.org/v4/components/search-input#with-submit-button)
+
+![search input with submit](./img/search-with-submit.png)
+
+ Adds a new search input variant that includes a submit button.
+
+### [New horizontal sub-navigation](https://www.patternfly.org/v4/components/navigation#horizontal-subnav)
+
+![page with vertical and horizontal sub-navigation](./img/horizontal-sub-nav.png)
+
+Introduces a new horizontal secondary navigation bar that will replace the legacy tertiary navigation to use for exposing local navigation within a page. The new navigation bar is better differentiated from horizontal tabs. See a [full page demo](https://www.patternfly.org/v4/components/navigation/react-demos#horizontal-subnav).
+
+### [Multi-select rows in a table](https://www.patternfly.org/v4/components/table#composable-selectable)
+Users can now use `shift+select` to select a range of rows in composable table component.
+
+### [Popover diagonal positioning](https://www.patternfly.org/v4/components/popover)
```

Comment:
Other components link to the specific example showing the update, can we change this to 

https://www.patternfly.org/v4/components/popover/#advanced

Reply:
That makes sense